src/test/store/profile-view.test.js

```diff
@@ -3281,6 +3281,60 @@ describe('traced timing', function() {
     expect(self[stack_C]).toBe(profile.meta.interval);
   });
 
+  it('computes traced timing for an inverted tree', function() {
+    const { profile } = getProfileFromTextSamples(`
+      0  1  5  6
+      A  A  A  C
+         B  B
+            C
+    `);
+    profile.meta.interval = 0.5;
+    // Inverted this tree looks like this:
+    //
+    // 0 1 5 6
+    // A B C C
+    //   A B
+    //     A
+
+    // The stack indexes appear in the order they were listed in the samples.
+    const stack_A = 0;
+    const stack_B = 1;
+    const stack_BA = 2;
+    const stack_C = 3;
+    const stack_CB = 4;
+    const stack_CBA = 5;
```

Comment:
I'm concerned that call node indexes will change after an inversion (I'm pretty sure they actually do?).  Moreover we really want call node indexes and not stack indexes, so I think this is a too big assumption to make that the indexed are like this. This may be ok for the tests in non-inverted trees, although if you do the work for this test it's easy to use it in previous tests too.

For the tests for the sidebar timings, I did something like this:
```js
const {
  profile,
  funcNamesDictPerThread: [{ A, B, C, D, E }],
} = getProfileFromTextSamples();

// invert tree

const stack_A = getCallNodeIndexFromPath(callNodeTable, [A]);
const stack_CBA = getCallNodeIndexFromPath(callNodeTable, [C, B, A]);
```
This is easier for the sidebar timings because the functions take a call node path directly. Maybe you could hide this in a setup function returning a querying function, to make it easier to query the results: `getRunningTimeForPath([C, B, A])`.

This is useful because this makes sure we get the right index.